contracts/core/Controller.sol

```diff
@@ -945,6 +985,9 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         // decrease amount of short otoken in liquidated vault, index of short otoken to decrease is hardcoded at 0
         vaults[_args.owner][_args.vaultId].removeShort(vault.shortOtokens[0], _args.amount, 0);
 
+        // decrease internal naked margin collateral amount
+        nakedPoolBalance[vault.collateralAssets[0]] = nakedPoolBalance[vault.collateralAssets[0]].sub(collateralToSell);
+
```

Comment:
this should only happen if vaultType = 1 right?

Reply:
never mind, this is under liquidation 😄

Reply:
Yes, _liquidate function will only work if vaultType is 1 anyway as this will revert if it is not https://github.com/opynfinance/GammaProtocol/blob/3ee964999ff7642251d1db3efeb4dd2336d5c59d/contracts/core/Controller.sol#L961